courses/free-courses-en.md

```diff
@@ -726,6 +726,7 @@
 * [PyTorch tutorials by PyTorch.org](https://pytorch.org/tutorials)
 * [Stanford SEE 229 - Machine Learning](https://see.stanford.edu/Course/CS229)
 * [Stanford University Machine Learning](https://www.coursera.org/learn/machine-learning)
+* [Machine Learning Specialization] (https://www.coursera.org/specializations/machine-learning-introduction)
```

Comment:
Add, if possible, the instructors and platform...

```suggestion
* [Machine Learning Specialization] (https://www.coursera.org/specializations/machine-learning-introduction) - Andrew Ng, Eddy Shyu, Aarti Bagul, Geoff Ladwig ( Coursera)
```

Reply:
`Coursera` note should go without spaces. check the difference between this and debc9d7639ff9cdbe0b623bdb2ac43117c56b3e9 😉

Reply:
> Reopened...
> 
> `Coursera` note should go without spaces. check the difference between suggestion and [debc9d7](https://github.com/EbookFoundation/free-programming-books/commit/debc9d7639ff9cdbe0b623bdb2ac43117c56b3e9) 😉

Resolved by commit SHA 336ddd5a8eca45dc6c900d85e2620ae3acfa0493